rootfs/etc/nginx/lua/balancer/ewma.lua

```diff
@@ -0,0 +1,128 @@
+-- Original Authors: Shiv Nagarajan & Scott Francis
+-- Accessed: March 12, 2018
+-- Inspiration drawn from:
+-- https://github.com/twitter/finagle/blob/1bc837c4feafc0096e43c0e98516a8e1c50c4421/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/PeakEwma.scala
+
+
+local resty_lock = require("resty.lock")
+local util = require("util.util")
+local split = require("util.split")
```

Comment:
you can just put whatever the functions needed from `util/split.lua` into `util.lua`, I think it is only `split_upstream_var` and `get_first_value`